Whiskey/Functions/Find-WhiskeyPowerShellModule.ps1

```diff
@@ -41,46 +41,119 @@ function Find-WhiskeyPowerShellModule
     Set-StrictMode -Version 'Latest'
     Use-CallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
 
-    # If you want to upgrade the PackageManagement and PowerShellGet versions, you must also update:
-    # * appveyor.yml
-    # * PublishPowerShellModule task
-    Import-Module -Name 'PackageManagement' -RequiredVersion '1.4.7' -Global
-    Import-Module -Name 'PowerShellGet' -RequiredVersion '2.2.5' -Global
-
-    Write-WhiskeyDebug -Message ('{0}  {1} ->' -f $Name,$Version)
-    if( $Version )
+    Write-WhiskeyDebug '\Find-WhiskeyPowerShellModule\' -Indent
+
+    function Import
     {
-        $atVersionString = ' at version {0}' -f $Version
+        [CmdletBinding()]
+        param(
+            [Parameter(Mandatory)]
+            [String] $Name,
+
+            [Parameter(Mandatory)]
+            [Version] $MinimumVersion,
 
-        if( -not [Management.Automation.WildcardPattern]::ContainsWildcardCharacters($version) -and [Version]::TryParse($Version,[ref]$null) )
+            [Parameter(Mandatory)]
+            [Version] $MaximumVersion
+        )
+
+        Write-WhiskeyDebug "Importing module $($Name) $($MinimumVersion) <= $($MaximumVersion)."

```

Comment:
Perhaps a little different formatting, here are assorted output thoughts, first is current: 

   Importing module X 1.2.3 <= 2.3.4
   Importing module X for version 1.2.3 - 2.3.4
   Importing module X 1.6.7 (>= 1.2.3, <= 2.3.4) -- after version resolved

Reply:
I'll update to be the second option, so it's clearer it's a range.

We can't add the resolved version because then we wouldn't be able to have this debug message here.